src/templates/default/reviews/talk_proposal_list.html

```diff
@@ -49,7 +49,7 @@ <h4>{% trans 'Personal Review Stats' %}</h4>
 <!-- Nav tabs -->
 <ul class="nav nav-tabs reviews-tabs js-hash-tabs" role="tablist">
 	<li role="presentation" class="active"><a href="#unreviewed" aria-controls="unreviewed" role="tab" data-toggle="tab">{% trans 'Unreviewed Talk Proposals' %} ({{ object_list|length }})</a></li>
-	<li role="presentation"><a href="#reviewed" aria-controls="reviewed" role="tab" data-toggle="tab">{% trans 'Reviewed Talk Proposals' %} ({{ reviews|length }})</a></li>
+	<li role="presentation"><a href="#reviewed" aria-controls="reviewed" role="tab" data-toggle="tab">{% trans 'Reviewed Talk Proposals' %} ({{ stage_1_reviews.count|add:stage_2_reviews.count}})</a></li>
```

Comment:
This should use `length` instead. Let’s calculate the value in the view, this is too long to write in the template.

Reply:
@uranusjr, 
Thank you for the advice! I made some adjustments in this commit. [6130d89](https://github.com/pycontw/pycon.tw/pull/1068/commits/6130d896532fadbe393cd703afb6ee8cc32541e6).

Also, instead of returning None, I changed the return value to make sure the query is still working on stage 1.